### DISCUSSION TITLE
[Question]:  How to fit the camera in the layout without changing resolution?

### DISCUSSION THREAD

**Discussion by HHUBSS** [blakeblackshear/frigate#13476] [Ask A Question] [answered]
### What is your question?

How to remove the black in the sides in the camera alert without changing the resolution? I think this is fit for 720p but my camera is set to 640p. How to fit this inside so I can maximize the space? 
![image](https://github.com/user-attachments/assets/e9adb1dd-bb55-446e-9483-9be72fa659dd)

**Comment by hawkeye217** [comment]
See the [FAQ](https://github.com/blakeblackshear/frigate/discussions/12724), Usage question 4.

You'd have to change your resolution.

**Comment by NickM-27** [comment, marked as answer]
You can't, your camera is not a 16:9 resolution so it won't fit perfectly into the box.

**Reply by HHUBSS** [reply]
Thank you. I will adjust my camera then.
